tests/sanity/tests/model/recruiting/talents-page.ts

```diff
@@ -59,6 +59,8 @@ export class TalentsPage extends CommonRecruitingPage {
   openOtherSkills = (): Locator => this.page.getByText('Other')
   skillsLink = (): Locator => this.page.locator('text=Skills')
   newSkillButton = (): Locator => this.page.locator('button:has-text("Skill")')
+  emailContact = (): Locator =>
+    this.page.locator('div[class^="popupPanel-body"] div.horizontal button[id="gmail:string:Email"]')
```

Comment:
`.horizontal` is generic class in our Scroller component, so I think it would be better to use `.popupPanel-body__header` (or another class that is used only in the header) instead of `.horizontal`. It'll make sure that we are trying to check email in the header and not in any other place in the panel.

Reply:
I don't think it should be a problem, since horizontal is looking that exact same area.
<img width="1087" alt="image" src="https://github.com/hcengineering/platform/assets/167111741/376211f9-ebc9-4d9d-9ed4-c6a7c2f4eea5">

but since you know components a lot better than I do, I will make an update